querydsl-core/src/main/java/com/querydsl/core/util/JavaSyntaxUtils.java

```diff
@@ -27,63 +25,8 @@
 
     private JavaSyntaxUtils() { }
```

Comment:
Why not delete this class, and just call `SourceVersion.isKeyword`? This is just a static utility class that wraps a call to a static method with the same signature.